elements/pfe-page-status/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@patternfly/pfe-page-status",
+  "pfelement": {
+    "className": "PfePageStatus",
+    "elementName": "pfe-page-status"
+  },
+  "version": "1.0.0-prerelease.13",
+  "description": "Page element for PatternFly Elements",
+  "repository": {
+    "type": "git",
+    "url": ""
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "index.js",
+  "scripts": {
+    "build": "../../node_modules/.bin/gulp && ../../node_modules/.bin/prettier --ignore-path ../../.prettierignore --write '**/*.{js,json}'",
+    "dev": "../../node_modules/.bin/gulp dev",
+    "test": "../../node_modules/.bin/wct --configFile ../../wct.conf.json elements/pfe-page-status/test/"
+  },
+  "author": {
```

Comment:
In other packages we have this for you Mark, is that okay:
```
"contributors": [
    {
      "name": "Mark Caron",
      "email": "mark@redhat.com",
      "url": "https://github.com/markcaron"
    }
  ],
```

Reply:
Sure. I'll replace "author"